contracts/starknet/src/escrow.cairo

```diff
@@ -236,25 +236,30 @@ mod Escrow {
     fn claim_payment(
         ref self: ContractState,
         from_address: felt252,
-        order_id: u256,
-        recipient_address: EthAddress,
-        amount: u256
+        order_id: felt252,
```

Comment:
Why did it change from u256 to felt252?

Reply:
Also, you create u256 variables underneath

Reply:
The way to receive a message from L1 in L2 is by sending an array, if this array is not a felt252, Cairo tries to typecast it to felt, and it is not space-efficient.

Receiving u256 in L2 was the reason why the message sent from L1 was not space efficient, sending an array of u256 but only filling each element with u128 values. 

Thus, the correct way to receive a message from L1 in SN is to receive felt values, and then casting them to whichever type is needed inside the function. This is how now it is possible to send an array of only 3 u256.

Reply:
I have reverted this change